fakebook/src/app/models/user.ts

```diff
@@ -0,0 +1,10 @@
+export interface User {
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string | undefined;
+  profilePictureUrl: string | null;
+  status: string | undefined;
+  birthDate: Date;
```

Comment:
Are users required to supply their birthdate? If not, should this be `birthDate: Date | undefined`?